AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/AnyModelIntegrationTests.swift

```diff
@@ -8,10 +8,8 @@
 import XCTest
 import Foundation
 import AWSAPIPlugin
-import AWSPluginsCore
```

Comment:
Uses of AWSPluginsCore has been removed since we do not provide that as a public facing package. All functionality in AWSPluginsCore is provided to the plugins to use. There are some types that should be moved to the plugin, such as AnyModel to DataStore. The sync based tests should also be moved over to DataStore's integration test targets, which will take on DataStore as a dependency, including the types that it has (AnyModel, MutationSync)